src/core/public/apm_system.test.ts

```diff
@@ -8,12 +8,12 @@
 
 jest.mock('@elastic/apm-rum');
 import type { DeeplyMockedKeys, MockedKeys } from '@kbn/utility-types-jest';
+import { executionContextServiceMock } from './mocks';
```

Comment:
NIT: import from mock package (will be necessary when this file is moved to a package too)